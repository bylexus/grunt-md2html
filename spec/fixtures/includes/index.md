<h1> Index </h1>
<% print(grunt.file.read('spec/fixtures/includes/_inc1.md')) %>
<div><% print(grunt.file.read('spec/fixtures/includes/_inc2.md').trim()) %></div>
