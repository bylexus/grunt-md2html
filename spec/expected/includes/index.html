<h1> Index </h1>
__Inc1__

<div>Source:spec/fixtures/includes/index.md</div>
